internal/broker/bind_create_test.go

```diff
@@ -272,6 +275,58 @@ func TestCreateBindingEndpoint(t *testing.T) {
 		}`, fixture.PlanId, customExpirationSeconds), t)
 		require.Equal(t, http.StatusBadRequest, response.StatusCode)
 	})
+
+	t.Run("should return 404 for not existing binding", func(t *testing.T) {
+		// given
+		instanceID := "1"
+		bindingID := uuid.New().String()
+		endpoint := fmt.Sprintf("v2/service_instances/%s/service_bindings/%s?accepts_incomplete=false", instanceID, bindingID)
+
+		// when
+		response := CallAPI(httpServer, "GET", endpoint, "", t)
```

Comment:
This is kind of different case than just not existing binding. This is case of empty `bindingID`.
Certainly test case is valid but the description seem to be inappropriate.
Please create test case with not existing but valid `bindingID`.